### ISSUE TITLE
Cant get HA working directly

### ISSUE THREAD

**Issue by sh00t2kill** [AntoineGS/teams-status-rs#9]
I have this working via MQTT so i know the app / teams part is working, but I am getting nothing in HA (other than my MQTT entities)

I cant see anything in the HA log that would suggest an issue.
I know my token works, i have tested it manually.

Is there a specific format the token has to be in?

**Comment by AntoineGS** [comment]
Hmm do the logs `output.log` in the application show anything of value?
I have not spent much time working on logging but there are some so we might get lucky.
As far as a format, just the string, it will encrypt it on first load and replace it in the configuration file with the encrypted version.

**Comment by sh00t2kill** [comment]
No, nothing in the output log, just JSON strings.
I will try adding it again just to be sure.

**Comment by AntoineGS** [comment]
Just had a thought, the application will prioritize the MQTT integration over HA, and will not do both at the same time. 
So if you still have MQTT configurations in place the HA will not be used. 
Could this be it?

**Comment by sh00t2kill** [comment]
Could be it. I'll disable mqtt and see how I go.

**Comment by sh00t2kill** [comment]
I removed all the mqtt settings, updated the token, and tried again.

I get an entry in the log reading teams info, but i dont get an icon in the taskbar, so im guessing it didnt start correctly.

I can see the token got encrypted in the config file, and the basic mqtt settings with no values back in the config file.


Turns out i forgot to add the port to my HA url; Oops! I added that and now its working!

Thanks for your help even though it was my problem.

Id suggest probably some logging of errors though, especially if its a "cant connect so im going to quit" type error, as i went down a rabbit hole of trying to work out how to quit the app --- the reason i couldnt find it was because it wasnt running, but the logs didnt actually say it had quit.
